### DISCUSSION TITLE
Can we disable Splash Screen in Mac?

### DISCUSSION THREAD

**Discussion by Dran9** [espanso/espanso#1545] [Q&A]
Hi, is it possible to disable splash screen every time mac starts?

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

You can set `show_notifications: false` in your `$CONFIG/config/default.yml` file, as documented [here](https://espanso.org/docs/configuration/options/#options-reference).

Is that what you mean?

**Comment by Dran9** [comment]
Hi Stephen thanks for your answer I mean this window every time I initiate a session in my Mac. I’d prefer to have that disabled but can’t find a way. Other notifications are ok.

Thanks and I hope we will soon see an update of your great app. 

Daniel

￼

**Comment by smeech** [comment]
If the setting doesn't remove the splash screen, then I would call that a bug. I'll flag it as such.

> Thanks and I hope we will soon see an update of your great app.

Not mine, I'm afraid! I'm just trying to help in a small way. There is now a team supporting Federico and the program, and a [Discord server](https://discord.gg/wHjeTySG), if you're interested. An update _is_ due soon.

**Comment by olitreadwell** [comment]
@federico-terzi is there a way to disable the splash screen on macos? 

It looks like this was also asked about here https://www.reddit.com/r/espanso/comments/yvs9nm/how_can_i_disable_the_espanso_is_already_running/

<img width="805" height="649" alt="image" src="https://github.com/user-attachments/assets/2ff77cce-2f66-4661-aeff-b072c53a4443" />
